### PR TITLE
Fixed null reference exception when processing the Pause order

### PR DESCRIPTION
**Pull request by Mailaender**
Discovered by Coverity.

### CODE REVIEW

File: OpenRA.Game/Network/UnitOrders.cs [hunk 1]
```diff
@@ -117,7 +117,7 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 						if (client != null)
 						{
 							var pause = order.TargetString == "Pause";
-							if (orderManager.World.Paused != pause && !world.LobbyInfo.IsSinglePlayer)
+							if (orderManager.World.Paused != pause && world != null && !world.LobbyInfo.IsSinglePlayer)
```
Comment (pchote) [inline OpenRA.Game/Network/UnitOrders.cs]: This looks bogus - you're mixing `orderManager.World` and `world`.

Comment (Mailaender) [inline OpenRA.Game/Network/UnitOrders.cs]: I also thought about that, but we are using `world` in the methods above, too.

Comment (Mailaender) [inline OpenRA.Game/Network/UnitOrders.cs]: > you're mixing `orderManager.World` and `world`.

Not me, this was actually introduced by yourself in https://github.com/OpenRA/OpenRA/pull/2991 so I assume you knew what you were doing back then. If you spotted a mistake of your own, you need to elaborate.

**Comment by abcdefg30** [comment]
:+1: / :white_check_mark:

**Comment by penev92** [comment]
Well this silences ReSharper and Coverity, so :+1: 
The `world` that is passed to the method as argument is used in a bunch of other places, while `orderManager.World` is used only here, so `¯\_(ツ)_/¯`

**Pull request merged**